tripy/tests/backend/test_compiler_api.py

```diff
@@ -101,7 +101,8 @@ def single_return_executable():
 @pytest.fixture(scope="session")
 def multiple_return_executable():
     compiler = tp.Compiler(returns_multiple_tensors)
-    return compiler.compile(tp.InputInfo((2, 2), dtype=tp.float32), tp.InputInfo((2, 2), dtype=tp.float32))
+    # TODO: revert back to InputInfo (2,2) when https://github.com/NVIDIA/TensorRT-Incubator/issues/44 is fixed.
```

Comment:
Is this issue number right?

Reply:
Yes, not sure why github does not initialize issues from 1.

Reply:
Huh apparently the IDs are unique across PRs and issues. 43 PRs before you created that issue, hence it became #44.